<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="weqw">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <property name="horizontalSpacing">
     <number>-1</number>
    </property>
    <item row="1" column="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_large"/>
   </widget>
   <widget class="QMenu" name="menuScenes">
    <property name="title">
     <string>Scenes</string>
    </property>
    <addaction name="actionScene_1"/>
    <addaction name="actionScene_2"/>
    <addaction name="actionScene_3"/>
    <addaction name="actionScene_4"/>
    <addaction name="actionScene_5"/>
    <addaction name="actionScene_6"/>
    <addaction name="actionScene_7"/>
    <addaction name="actionScene_844"/>
    <addaction name="separator"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuRay_Tracer">
    <property name="title">
     <string>Ray Tracer</string>
    </property>
    <addaction name="actionBump_Mapping"/>
    <addaction name="actionShadowing"/>
    <addaction name="actionReflection"/>
    <addaction name="actionSuperSampling"/>
    <addaction name="actionInterception"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <widget class="QMenu" name="menuTransform">
     <property name="title">
      <string>Transform</string>
     </property>
     <addaction name="actionCameraMoveLeft"/>
     <addaction name="actionCameraMoveRight"/>
     <addaction name="actionCameraMoveUp"/>
     <addaction name="actionCameraMoveDown"/>
     <addaction name="separator"/>
     <addaction name="actionCameraMoveForward"/>
     <addaction name="actionCameraMoveBackward"/>
     <addaction name="separator"/>
     <addaction name="actionCameraTransform"/>
    </widget>
    <addaction name="menuTransform"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_IN"/>
    <addaction name="actionZoom_OUT"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Camera"/>
   </widget>
   <widget class="QMenu" name="menuOptimization">
    <property name="title">
     <string>Optimization</string>
    </property>
    <addaction name="actionMulti_Threading"/>
    <addaction name="actionBlock"/>
   </widget>
   <widget class="QMenu" name="menuObjects">
    <property name="title">
     <string>Objects</string>
    </property>
    <widget class="QMenu" name="menuX_plane">
     <property name="title">
      <string>X-plane</string>
     </property>
     <addaction name="actionXPlaneLeft"/>
     <addaction name="actionXPlaneRight"/>
    </widget>
    <widget class="QMenu" name="menuY_Plane">
     <property name="title">
      <string>Y-Plane</string>
     </property>
     <addaction name="actionYPlane_Plus_10_degrees"/>
     <addaction name="actionYPlane_Minus_10_degrees"/>
    </widget>
    <widget class="QMenu" name="menuZ_Plane">
     <property name="title">
      <string>Z-Plane</string>
     </property>
     <addaction name="actionZPlane_Plus_10_degrees"/>
     <addaction name="actionZPlane_Minus_10_degrees"/>
    </widget>
    <addaction name="menuX_plane"/>
    <addaction name="menuY_Plane"/>
    <addaction name="menuZ_Plane"/>
    <addaction name="separator"/>
    <addaction name="actionScale"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuObjects"/>
   <addaction name="menuScenes"/>
   <addaction name="menuRay_Tracer"/>
   <addaction name="menuCamera"/>
   <addaction name="menuOptimization"/>
  </widget>
  <action name="actionScene_1">
   <property name="text">
    <string>Scene 1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionScene_2">
   <property name="text">
    <string>Scene 2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionScene_3">
   <property name="text">
    <string>Scene 3</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionScene_4">
   <property name="text">
    <string>Scene 4</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionScene_5">
   <property name="text">
    <string>Scene 5</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionScene_6">
   <property name="text">
    <string>Scene 6</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="actionBump_Mapping">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bump Mapping</string>
   </property>
   <property name="shortcut">
    <string>Meta+R, Meta+B</string>
   </property>
  </action>
  <action name="actionShadowing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shadowing</string>
   </property>
   <property name="shortcut">
    <string>Meta+R, Meta+S</string>
   </property>
  </action>
  <action name="actionScene_7">
   <property name="text">
    <string>Scene 7</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+7</string>
   </property>
  </action>
  <action name="actionScene_844">
   <property name="text">
    <string>Scene 8</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
  <action name="actionReset_Camera">
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionZoom_IN">
   <property name="text">
    <string>Zoom IN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_OUT">
   <property name="text">
    <string>Zoom OUT</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionCameraMoveLeft">
   <property name="text">
    <string>Move Left</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionCameraMoveRight">
   <property name="text">
    <string>Move Right</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionCameraMoveUp">
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="shortcut">
    <string>Alt+Up</string>
   </property>
  </action>
  <action name="actionCameraMoveDown">
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="shortcut">
    <string>Alt+Down</string>
   </property>
  </action>
  <action name="actionCameraMoveForward">
   <property name="text">
    <string>Move Forward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+Up</string>
   </property>
  </action>
  <action name="actionCameraMoveBackward">
   <property name="text">
    <string>Move Backward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+Down</string>
   </property>
  </action>
  <action name="actionCameraTransform">
   <property name="text">
    <string>Transform...</string>
   </property>
   <property name="shortcut">
    <string>Meta+T</string>
   </property>
  </action>
  <action name="actionMulti_Threading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Multi Threading</string>
   </property>
   <property name="shortcut">
    <string>Meta+O, Meta+M</string>
   </property>
  </action>
  <action name="actionSave_large">
   <property name="text">
    <string>Save large</string>
   </property>
   <property name="toolTip">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action001">
   <property name="text">
    <string>001</string>
   </property>
  </action>
  <action name="actionBlock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Block</string>
   </property>
   <property name="shortcut">
    <string>Meta+O, Meta+B</string>
   </property>
  </action>
  <action name="actionReflection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reflection</string>
   </property>
   <property name="shortcut">
    <string>Meta+R, Meta+R</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionDddd">
   <property name="text">
    <string>dddd</string>
   </property>
  </action>
  <action name="actionABC">
   <property name="text">
    <string>ABC</string>
   </property>
  </action>
  <action name="actionAsdasdasdas">
   <property name="text">
    <string>asdasdasdas</string>
   </property>
  </action>
  <action name="actionDef">
   <property name="text">
    <string>def</string>
   </property>
  </action>
  <action name="actionSss">
   <property name="text">
    <string>sss</string>
   </property>
  </action>
  <action name="actionWwww">
   <property name="text">
    <string>wwww</string>
   </property>
  </action>
  <action name="actionSs">
   <property name="text">
    <string>ss</string>
   </property>
  </action>
  <action name="actionSeeee">
   <property name="text">
    <string>gsdgfdgf</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionXPlaneLeft">
   <property name="text">
    <string>10 degrees</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionXPlaneRight">
   <property name="text">
    <string>-10 degrees</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionYPlane_Plus_10_degrees">
   <property name="text">
    <string>10 degrees</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionYPlane_Minus_10_degrees">
   <property name="text">
    <string>-10 degrees</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionZPlane_Plus_10_degrees">
   <property name="text">
    <string>10 degrees</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Left</string>
   </property>
  </action>
  <action name="actionZPlane_Minus_10_degrees">
   <property name="text">
    <string>-10 degrees</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Right</string>
   </property>
  </action>
  <action name="actionSuperSampling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SuperSampling</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionInterception">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interception</string>
   </property>
   <property name="shortcut">
    <string>Meta+R, Meta+I</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
